trivy.yaml

```diff
@@ -0,0 +1,4 @@
+exit-code: 2
```

Comment:
What's the goal of returning 2 ?

Reply:
Thanks for spotting, I cherry picked the wrong commit :) The `exit-code` needs to be `0`, I was testing earlier to see whether the config overrides what's in the github action for trivy. Thanks!